### PR TITLE
Add all menus slash command, beautify Mizza, send menu in last call

### PR DESCRIPTION
**Pull request by kubik369**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -94,55 +93,25 @@ export function startExpress() {
...
-        .status(200)
-        .send(`\`\`\`${getTodaysMizzaMenu(body)}\`\`\``);
-      curl.close();
-    });
-    curl.on('error', () => {
-      res.status(500).send();
-      curl.close();
-    });
-    curl.perform();
+  app.get('/menupresto', async (req, res) => {
```
Comment (tamasbogi) [inline src/routes.js:96]: what about error handling?

File: src/utils.js [hunk 2]
```diff
@@ -310,7 +311,26 @@ function getDayForMenu() {
...
+  try {
+    const body = await request(link);
+    return `${block}${parseMenu(body, allergens)}${block}`;
+  } catch (e) {
+    logger.error(e);
+    return `${block}Chyba počas načítavania menu :disappointed:${block}`;
+  }
+}
+
+export async function getAllMenus(allergens) {
```
Comment (tamasbogi) [inline src/utils.js:325]: do the requests in parallel
`const [presto, veglife, mizza] = await Promise.all([getMenu(...), getMenu(..), getMenu(...)])`

File: src/routes.js (hunk 1 above)
Comment (kubik369) [reply src/routes.js:96]: This should not throw errors.

File: src/utils.js [hunk 3]
```diff
@@ -333,19 +353,23 @@ export function getTodaysPrestoMenu(menu) {
   return menu;
 }
 
-export function getTodaysVeglifeMenu(menu) {
+export function parseTodaysVeglifeMenu(menu) {
   const slovakDays = ['', 'PONDELOK', 'UTOROK', 'STREDA', 'ŠTVRTOK', 'PIATOK', 'SOBOTA'];
   const today = getDayForMenu();
-
-  return menu
+  menu = menu
```
Comment (tamasbogi) [inline src/utils.js]: its generally a bad idea to modify the params
use another replace with regex like: `/+ Pestra.+$/`

Comment (kubik369) [reply src/utils.js]: Strings are immutable, so it should be all right.

**Review by tamasbogi — APPROVED** [review]

**Pull request merged**